packages/environment-check/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "@sap-ux/environment-check",
+    "version": "0.0.1",
+    "description": "SAP Fiori environment check",
+    "license": "Apache-2.0",
+    "bin": {
+        "envcheck": "bin/envcheck"
+    },
+    "main": "dist/index.js",
+    "private": false,
+    "scripts": {
+        "envcheck": "./bin/envcheck",
+        "build": "rimraf dist && tsc",
+        "clean": "rimraf dist test/test-output coverage",
+        "watch": "tsc --watch",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors",
+        "test-u": "jest --ci --forceExit --detectOpenHandles --colors -u",
+        "link": "pnpm link --global",
+        "unlink": "pnpm unlink --global"
+    },
+    "dependencies": {
+        "@sap/bas-sdk": "^2.1.0",
+        "@sap-ux/axios-extension": "workspace:*",
+        "@sap-ux/btp-utils": "workspace:*",
+        "@sap-ux/logger": "workspace:*",
+        "@sap-ux/ui5-config": "workspace:*",
+        "@types/archiver": "5.1.1",
+        "archiver": "5.3.0",
+        "axios": "0.24.0",
+        "findit2": "2.2.3",
+        "i18next": "20.3.2",
+        "minimist": "^1.2.6",
```

Comment:
Please use fixed version number (same for `@sap/bas-sdk`)

Reply:
Updated